packages/grafana-ui/src/components/Legend/Legend.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { StatID } from '../../utils/statsCalculator';
+import { DisplayValue } from '../../types/index';
+
+export interface LegendBasicOptions {
+  isVisible: boolean;
+  asTable: boolean;
+}
+
+export interface LegendRenderOptions {
+  placement: LegendPlacement;
+  hideEmpty?: boolean;
+  hideZero?: boolean;
+}
+
+export type LegendPlacement = 'under' | 'right' | 'over'; // Over used by piechart
+
+export interface LegendOptions extends LegendBasicOptions, LegendRenderOptions {
+  stats?: StatID[];
+  decimals?: number;
```

Comment:
Why are decimals necessary?  If using DisplayValue the `text` property will have already formatted the value.

Reply:
Current implementation of legend allows decimals config for stats:

![image](https://user-images.githubusercontent.com/2376619/55970814-8f39a600-5c80-11e9-8bb9-8ef30c1ad02b.png)

Reply:
As we just talked @ryantxu, `stats` and `decimals` are going to be removed from LegendOptions as these ain't Legend specific

Reply:
👍 -- the decimals should be in the *panel*, not the legend component.